core/src/utilities.zig

```diff
@@ -188,6 +189,60 @@ pub fn Slice_Vector(comptime Slice: type) type {
     };
 }
 
+pub fn max_enum_tag(T: type) @typeInfo(T).Enum.tag_type {
+    var max_tag: @typeInfo(T).Enum.tag_type = 0;
```

Comment:
Could maybe do something like
```zig
        if (@typeInfo(T) != .Enum) {
            @compileError("max_enum_tag expects an enum type");
        }
```
Also, is it possible that the enum has a max value lower than 0? If so, then this would incorrectly think it's 0. We could check and iensure that there is at least 1 enum field, and initialize the max tag to the value of the first.